### PR TITLE
docs: update polish translation

### PR DESCRIPTION
**Pull request by PiotrekPKP**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I performed a functional test on my final commit

---

## Changelog

- updated polish translation to match current english docs based on [this diff](https://github.com/t3-oss/create-t3-app/compare/a36a100e9aedad2f4ea0e70f63b44ff879b0faf8..next), comparing the commit which polish translation was introduced in with the latest one
- fixed `netlify.md` file use use polish language

💯

### CODE REVIEW

File: www/src/pages/pl/folder-structure.mdx [hunk 1]
```diff
@@ -47,7 +47,7 @@ Folder `pages` zawiera wszystkie strony aplikacji Next.js. Plik `index.tsx` w g
 
 #### `src/pages/api`
 
-Folder `api` zawiera wszystkie API route'y aplikacji Next.js. Plik `examples.ts` zawiera przykład takiego route'a korzystającego z funkcji [Next.js API route](https://nextjs.org/docs/api-routes/introduction) oraz z Prismy. Plik `restricted.ts` zawiera przykład route'a korzystającego z funkcji [Next.js API route](https://nextjs.org/docs/api-routes/introduction) i jest on chroniony ("authenticated") poprzez [NextAuth.js](https://next-auth.js.org/).
+Folder `api` zawiera wszystkie route'y twojej aplikacji Next.js. Po więcej informacji o api route'ach, zobacz [ich dokumentację od Next.jsa](https://nextjs.org/docs/api-routes/introduction).
```
Comment (matibox) [inline www/src/pages/pl/folder-structure.mdx:50]:
```suggestion
Folder `api` zawiera wszystkie route'y twojej aplikacji Next.js. Po więcej informacji o api route'ach, zobacz [ich dokumentację od Next.js](https://nextjs.org/docs/api-routes/introduction).
```

File: www/src/pages/pl/folder-structure.mdx [hunk 2]
```diff
@@ -75,7 +75,7 @@ Folder `server` używany jest do stworzenia wyraźnego podziału między kodem d
 
 ### `src/server/auth.ts`
 
-Zawiera narzędzia do uwierzytelniania, takie jak pobieranie sesji użytkownika po stronie serwera. Po więcej informacji, zobacz [korzystanie z NextAuth.js](/pl/usage/next-auth#korzystanie-z-trpc).
+Jest to główny punkt wejścia dla systemu authentication po stronie serwera. Ustawiamy tutaj [opcje konfiguracji](/pl/usage/next-auth), [dostosowujemy model użytkownika](/pl/usage/next-auth#inclusion-of-userid-on-the-session) oraz zapewniamy narzędzia DX dla authentication, takie jak funkcja pobierania sesji użytkownika po stronie serwera. Po więcej informacji, zobacz [korzystanie z NextAuth.js](/pl/usage/next-auth#korzystanie-z-trpc).
```
Comment (matibox) [inline www/src/pages/pl/folder-structure.mdx:78]:
```suggestion
Jest to główny punkt wejścia dla systemu uwierzytelniania po stronie serwera. Ustawiamy tutaj [opcje konfiguracji](/pl/usage/next-auth), [dostosowujemy model użytkownika](/pl/usage/next-auth#inclusion-of-userid-on-the-session) oraz zapewniamy narzędzia DX dla uwierzytelniania, takie jak funkcja pobierania sesji użytkownika po stronie serwera. Po więcej informacji, zobacz [korzystanie z NextAuth.js](/pl/usage/next-auth#korzystanie-z-trpc).
```

File: www/src/pages/pl/usage/env-variables.md [hunk 3]
```diff
@@ -5,46 +5,49 @@ layout: ../../../layouts/docs.astro
...
+  // NEXT_PUBLIC_CLIENTVAR: process.env.NEXT_PUBLIC_CLIENTVAR,
+};
 ```
 
 ### Schemat Dla Serwera
 
 Zdefiniuj tutaj zmienne środowiskowe dla serwera.
 
-Koniecznie **nie** prefixuj tutejszych kluczy `NEXT_PUBLIC_`. Jeżeli to zrobisz, walidacja nie zadziała, pomagając ci w wykryciu niewłaściwej konfiguracji.
+Koniecznie **nie** prefixuj tutejszych kluczy `NEXT_PUBLIC_`, aby nie "zleakować" ich przypadkowo do klienta.
```
Comment (matibox) [inline www/src/pages/pl/usage/env-variables.md:35]:
```suggestion
Koniecznie **nie** prefixuj tutejszych kluczy `NEXT_PUBLIC_`, aby przypadkiem nie ujawnić ich do klienta.
```

File: www/src/pages/pl/usage/env-variables.md [hunk 4]
```diff
@@ -5,46 +5,49 @@ layout: ../../../layouts/docs.astro
...
 
 ### Schemat Dla Klienta
 
 Zdefiniuj tutaj zmienne środowiskowe dla klienta.
 
 Aby ujawnić zmienne dla klienta dodaj prefix `NEXT_PUBLIC`. Jeżeli tego nie zrobisz, walidacja nie zadziała, pomagając ci w wykryciu niewłaściwej konfiguracji.
 
+### Obiekt `processEnv`
+
+Wykorzystaj ("destructuj") tutaj obiekt `process.env`.
```
Comment (matibox) [inline www/src/pages/pl/usage/env-variables.md:45]:
```suggestion
Wykorzystaj destrukturyzację obiektu `process.env`.
```

File: www/src/pages/pl/usage/env-variables.md [hunk 5]
```diff
@@ -5,46 +5,49 @@ layout: ../../../layouts/docs.astro
...
+
+Potrzebny jest nam obiekt, który parse'ować możemy z naszymi schematami Zoda, a z powodu sposobu w jaki Next.js przetwarza zmienne środowiskowe, nie możesz destructować obiektu `process.env` tak jak zwykłego obiektu - trzeba to zrobić manualnie.
```
Comment (matibox) [inline www/src/pages/pl/usage/env-variables.md:47]:
```suggestion
Potrzebny jest nam obiekt, który parse'ować możemy z naszymi schematami Zoda, a z powodu sposobu w jaki Next.js przetwarza zmienne środowiskowe, nie możesz destrukturyzować obiektu `process.env` tak jak zwykłego obiektu - trzeba to zrobić manualnie.
```

File: www/src/pages/pl/usage/env-variables.md [hunk 6]
```diff
@@ -5,46 +5,49 @@ layout: ../../../layouts/docs.astro
...
+
+TypeScript zapewni poprawność destrukcji obiektu i zapobiegnie sytuacji, w której zapomnisz o jakimś kluczu.
```
Comment (matibox) [inline www/src/pages/pl/usage/env-variables.md:49]:
```suggestion
TypeScript zapewni poprawność destrukturyzacji obiektu i zapobiegnie sytuacji, w której zapomnisz o jakimś kluczu.
```

File: www/src/pages/pl/usage/trpc.md [hunk 7]
```diff
@@ -133,6 +95,74 @@ const UserPage = () => {
...
+}
+```
+
+## Pliki
+
+tRPC wymaga dużo boilerplate'u, który `create-t3-app` przygotowuje za Ciebie. Przejdźmy więc po kolei po plikach, które są generowane:
+
+### 📄 `pages/api/trpc/[trpc].ts`
+
+Jest to właściwy punkt początkowy dla twojego API - to on ujawnia dla reszty aplikacji twój router od tRPC. Prawdopodobnie nie będziesz musiał edytować tego pliku, ale jeżeli zajdzie taka potrzeba (np. do włączenia CORSa), warto wiedzieć o tym, iż eksportowany `createNextApiHandler` to [Next.js API handler](https://nextjs.org/docs/api-routes/introduction), który pobiera obiekt [zapytania](https://developer.mozilla.org/en-US/docs/Web/API/Request) i [odpowiedzi](https://developer.mozilla.org/en-US/docs/Web/API/Response) serwera. Oznacza to, iż możesz zawrzeć `createNextApiHandler` w middleware'rze, w jakim tylko chcesz. Poniżej znajdziesz [przykładowy kod](#aktywacja-cors), dzięki któremu dodasz CORS.
```
Comment (matibox) [inline www/src/pages/pl/usage/trpc.md:134]:
```suggestion
Jest to właściwy punkt początkowy dla twojego API - to on ujawnia dla reszty aplikacji twój router od tRPC. Prawdopodobnie nie będziesz musiał edytować tego pliku, ale jeżeli zajdzie taka potrzeba (np. do włączenia CORSa), warto wiedzieć o tym, iż eksportowany `createNextApiHandler` to [Next.js API handler](https://nextjs.org/docs/api-routes/introduction), który pobiera obiekt [zapytania](https://developer.mozilla.org/en-US/docs/Web/API/Request) i [odpowiedzi](https://developer.mozilla.org/en-US/docs/Web/API/Response) serwera. Oznacza to, iż możesz zawrzeć `createNextApiHandler` w middleware, w jakim tylko chcesz. Poniżej znajdziesz [przykładowy kod](#aktywacja-cors), dzięki któremu dodasz CORS.
```

File: www/src/pages/pl/usage/trpc.md [hunk 8]
```diff
@@ -133,6 +95,74 @@ const UserPage = () => {
...
+
+### 📄 `server/api/trpc.ts`
+
+Plik ten podzielony jest na dwie części - tworzenie kontekstu oraz inicjalizacji tRPC:
+
+1. Definiujemy kontekst przesyłany do procedur tRPC. Kontekt, to dane do których dostęp mają wszystkie twoje procedury tRPC. Jest to doskonałe miejsce do umieszczenia rzeczy, takich jak połączenia z bazą danych, informacje o uwierzytelnianiu, itp. W Create T3 App korzystamy z dwóch funkcji, aby umożliwić korzystanie z części kontekstu bez dostępu do obiektu zapytania.
```
Comment (matibox) [inline www/src/pages/pl/usage/trpc.md:140]:
```suggestion
1. Definiujemy kontekst przesyłany do procedur tRPC. Kontekst, to dane do których dostęp mają wszystkie twoje procedury tRPC. Jest to doskonałe miejsce do umieszczenia rzeczy, takich jak połączenia z bazą danych, informacje o uwierzytelnianiu, itp. W Create T3 App korzystamy z dwóch funkcji, aby umożliwić korzystanie z części kontekstu bez dostępu do obiektu zapytania.
```

File: www/src/pages/pl/usage/trpc.md [hunk 9]
```diff
@@ -133,6 +95,74 @@ const UserPage = () => {
...
+
+- `createInnerTRPCContext`: Tutaj definiujesz kontekst, który nie zależy od obiektu zapytania, np. połączenie z bazą danych. Możesz wykorzystać funkcję tą do [testów integracji](#przykładowy-test-integracji) oraz [funkcji pomocniczych SSG](https://trpc.io/docs/v10/ssg-helpers), gdzie nie posiadasz obiektu zapytania.
```
Comment (matibox) [inline www/src/pages/pl/usage/trpc.md:142]:
```suggestion
- `createInnerTRPCContext`: Tutaj definiujesz kontekst, który nie zależy od obiektu zapytania, np. połączenie z bazą danych. Możesz wykorzystać tą funkcję do [testów integracji](#przykładowy-test-integracji) oraz [funkcji pomocniczych SSG](https://trpc.io/docs/v10/ssg-helpers), gdzie nie posiadasz obiektu zapytania.
```

File: www/src/pages/pl/usage/trpc.md [hunk 10]
```diff
@@ -133,6 +95,74 @@ const UserPage = () => {
...
+
+- `createTRPCContext`: Tutaj definiujesz kontekst, który zależny jest od zapytania, np. sesja użytkownika. Otrzymujesz sesję korzystając z obiektu `opts.req` a następnie posyłasz ją do funkcji `createInnerTRPCContext` w celu utworzenia finalnego kontekstu.
```
Comment (matibox) [inline www/src/pages/pl/usage/trpc.md:144]:
```suggestion
- `createTRPCContext`: Tutaj definiujesz kontekst, który zależny jest od zapytania, np. sesja użytkownika. Otrzymujesz sesję korzystając z obiektu `opts.req`, a następnie posyłasz ją do funkcji `createInnerTRPCContext` w celu utworzenia finalnego kontekstu.
```

File: www/src/pages/pl/usage/trpc.md [hunk 11]
```diff
@@ -133,6 +95,74 @@ const UserPage = () => {
...
+
+2. Inicjalizujemy tRPC i definiujemy [procedury](https://trpc.io/docs/v10/procedures) oraz [middleware'y](https://trpc.io/docs/v10/middlewares). Umownie, nie powinieneś eksportować całego obiektu `t` a jedynie poszczególne procedury i middleware'y.
```
Comment (matibox) [inline www/src/pages/pl/usage/trpc.md:146]:
```suggestion
2. Inicjalizujemy tRPC i definiujemy [procedury](https://trpc.io/docs/v10/procedures) oraz [middleware](https://trpc.io/docs/v10/middlewares). Umownie, nie powinieneś eksportować całego obiektu `t` a jedynie poszczególne procedury i middleware.
```

**Review by matibox — COMMENTED** [review]
Nie jestem do końca pewny co do tego middleware zamiast middleware'ów itd. Ale wydaje mi się że spolszczanie tego na siłę dziwnie brzmi.

**Comment by PiotrekPKP** [comment]
Dzięki @matibox!

File: www/src/pages/pl/usage/env-variables.md [hunk 12]
```diff
@@ -54,21 +57,72 @@ const schema = z.object({
...
 
-## server.mjs & client.mjs
+### Logika Walidacji
+
+_Dla zainteresowanego czytelnika:_
+
+<details>
+<summary>Zaawansowane: Logika walidacji</summary>
+
+W zależności od środowiska (seerwer lub klient) walidujemy albo oba schematy, albo tylko schemat klienta. Oznacza to, iż nawet jeśli zmienne środowiskowe serwera nie będą zdefiniowane, nie zostanie wyrzucony błąd walidacji - możemy więc mieć jeden punkt odniesienia do naszych zmiennych.
```
Comment (michalbundyra) [inline www/src/pages/pl/usage/env-variables.md:67]:
```suggestion
W zależności od środowiska (serwer lub klient) walidujemy albo oba schematy, albo tylko schemat klienta. Oznacza to, iż nawet jeśli zmienne środowiskowe serwera nie będą zdefiniowane, nie zostanie wyrzucony błąd walidacji - możemy więc mieć jeden punkt odniesienia do naszych zmiennych.
```

**Comment by PiotrekPKP** [comment]
In case anyone's wondering the PR is ready to merge :)

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**